src/p11_rsa.c

```diff
@@ -474,28 +475,37 @@ static int RSA_meth_set_finish(RSA_METHOD *meth, int (*finish)(RSA *rsa))
 /*
  * Overload the default OpenSSL methods for RSA
  */
+static RSA_METHOD *pkcs11_rsa_meth_ops = NULL;
+
 RSA_METHOD *PKCS11_get_rsa_method(void)
 {
-	static RSA_METHOD *ops = NULL;
-
-	if (!ops) {
+	if (!pkcs11_rsa_meth_ops) {
 		alloc_rsa_ex_index();
-		ops = RSA_meth_dup(RSA_get_default_method());
-		if (!ops)
+		pkcs11_rsa_meth_ops = RSA_meth_dup(RSA_get_default_method());
+		if (!pkcs11_rsa_meth_ops)
 			return NULL;
-		RSA_meth_set1_name(ops, "libp11 RSA method");
-		RSA_meth_set_flags(ops, 0);
-		RSA_meth_set_priv_enc(ops, pkcs11_rsa_priv_enc_method);
-		RSA_meth_set_priv_dec(ops, pkcs11_rsa_priv_dec_method);
-		RSA_meth_set_finish(ops, pkcs11_rsa_free_method);
+		RSA_meth_set1_name(pkcs11_rsa_meth_ops, "libp11 RSA method");
+		RSA_meth_set_flags(pkcs11_rsa_meth_ops, 0);
+		RSA_meth_set_priv_enc(pkcs11_rsa_meth_ops, pkcs11_rsa_priv_enc_method);
+		RSA_meth_set_priv_dec(pkcs11_rsa_meth_ops, pkcs11_rsa_priv_dec_method);
+		RSA_meth_set_finish(pkcs11_rsa_meth_ops, pkcs11_rsa_free_method);
 	}
-	return ops;
-}
+	return pkcs11_rsa_meth_ops;
+}	
```

Comment:
Any particular reasons for this tabulator?

Reply:
ops was renamed to pkcs11_rsa_meth_ops. Since there is only one ops in the file, it did not need to be renamed.